docs/interactions/Application_Commands.md

```diff
@@ -10,20 +12,23 @@ Application commands are commands that an application can register to Discord. T
 
 ###### Application Command Structure
 
-| Field                      | Type                                                                                                                                           | Description                                                                                                          | Valid Types |
-|----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|-------------|
-| id                         | snowflake                                                                                                                                      | unique id of the command                                                                                             | all         |
-| type?                      | one of [application command type](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-types)                | the type of command, defaults `1` if not set                                                                         | all         |
-| application_id             | snowflake                                                                                                                                      | unique id of the parent application                                                                                  | all         |
-| guild_id?                  | snowflake                                                                                                                                      | guild id of the command, if not global                                                                               | all         |
```

Comment:
Why was this removed? I still see it present in my guild commands

Reply:
Hmmm not sure how that got removed. Adding it back in